tests/e2e/export.go

```diff
@@ -26,6 +31,59 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+type Type string
+
+const (
+	GcloudExporter Type = "gcloud"
+)
+
+type Exporter struct {
+	Type         Type
+	Cmds         []string
+	ResourceName string
+}
+
+func (e *Exporter) Run() ([]byte, error) {
+	args := []string{}
+	args = append(args, e.Cmds...)
+	args = append(args, e.ResourceName)
+
+	cmd := exec.Command("gcloud", args...)
+	var out bytes.Buffer
+	cmd.Stdout = &out
+
+	if err := cmd.Run(); err != nil {
+		return nil, fmt.Errorf("error from cmd.Run: %w, stdout: %q", err, out.String())
+	}
+
+	cleanOutput := regexp.MustCompile(`(?ms)^(Updates are available.*$|To take a quick.*$)`).ReplaceAll(out.Bytes(), nil)
+	return cleanOutput, nil
+}
+
+func exportResourceWithGcloud(h *create.Harness, obj *unstructured.Unstructured) []byte {
+	resourceID, _, _ := unstructured.NestedString(obj.Object, "spec", "resourceID")
+	if resourceID == "" {
+		resourceID = obj.GetName()
+	}
+
+	gvk := obj.GroupVersionKind()
+	switch gvk.GroupKind() {
+	case v1beta1.LoggingLogMetricGVK.GroupKind():
+		e := &Exporter{
+			Cmds:         []string{"logging", "metrics", "describe"},
```

Comment:
I guess this doesn't work with mockgcp?

Reply:
One more reason why I should redo this w the export func from #1768